### ISSUE TITLE
Bug when painting with two hands simultaneously

### ISSUE THREAD

**Issue by dmarcos** [aframevr/a-painter#246]
The strokes are drawn from one hand to the other instead of a separate stroke per hand.

**Comment by SF5** [comment]
that is a feature in my opinion this way you can use the program as with tilt brush and make surfaces

**Comment by dmarcos** [comment]
True, sometimes bugs become features but we did not implement it deliberately and probably UX can be improved 😄

**Comment by colinfizgig** [comment]
I agree it could be a feature, but it tends to be a bit rough and noisy.  Maybe if there was a way to smooth the strokes?   I like the gradient it creates as well.
